dspy/evaluate/evaluate.py

```diff
@@ -178,12 +179,13 @@ def process_item(example):
         logger.info(f"Average Metric: {ncorrect} / {ntotal} ({round(100 * ncorrect / ntotal, 1)}%)")
 
         if display_table:
-            # Rename the 'correct' column to the name of the metric object
-            metric_name = metric.__name__ if isinstance(metric, types.FunctionType) else metric.__class__.__name__
-            # Construct a pandas DataFrame from the results
-            result_df = self._construct_result_table(results, metric_name)
+            if importlib.util.find_spec("pandas") is not None:
```

Comment:
shall we log a warning here to inform people that displaying table fails because pandas is not found?